src/components/Category/CategoryBanner.tsx

```diff
@@ -8,26 +8,35 @@ import TokenList from 'decentraland-gatsby/dist/utils/dom/TokenList'
 import './CategoryBanner.css'
 
 const icons = {
-  [ProposalType.Catalyst]: require('../../images/icons/catalyst.svg'),
-  [ProposalType.POI]: require('../../images/icons/poi.svg'),
-  [ProposalType.BanName]: require('../../images/icons/ban-name.svg'),
-  [ProposalType.Grant]: require('../../images/icons/grant.svg'),
-  [ProposalType.Poll]: require('../../images/icons/poll.svg'),
+  [ProposalType.Catalyst]: require("../../images/icons/catalyst.svg"),
+  [ProposalType.POI]: require("../../images/icons/poi.svg"),
+  [ProposalType.AddPOI]: require("../../images/icons/add-poi.svg"),
```

Comment:
Since the proposals add/remove poi should still be  a `ProposalType.POI` , I don't think adding new proposal type is necessary. 

Maybe what we are missing here is a new `PoiType` type, that could be the key for this Icons?